interface/reports/edit_dictation.php

```diff
@@ -0,0 +1,50 @@
+<!-- Form generated from formsWiz -->
+<?php
```

Comment:
Aren't you able to use the standard edit dictation form script?

Reply:
It is possible. I started this script some time ago and I know more now than I did then. It is possible. When I was updating the form, I thought the same thing. I just need to figure out what to do. I did it in another form by passing the encounter information and pulled the right form.